reviews/layout/helpers.py

```diff
@@ -13,7 +13,8 @@
 from reviews.datasource import PullRequest
 
 
-def render_pull_request_table(title: str, pull_requests: List[PullRequest]) -> Table:
+def render_pull_request_table(title: str, pull_requests: List[PullRequest],
```

Comment:
You will need to run linting for the code in both files to conform to [black](https://black.readthedocs.io/en/stable/) and pass CI.

more information on running linting can be found here: https://github.com/apoclyps/reviews#linting

Reply:
I'm not all too familiar with docker, but I'll try to get that set up and running and see if I could resolve that -- I'm currently just running off the terminal from my linux distro (Manjaro). Let me see if I could get a few alternatives running and see If I could click on the links.

Thanks for your help!

Reply:
The following are the dependencies you will need for linting: you can install them from the dev requirements file

```bash
pip install -r requirements_dev.txt
```

or install the individual versions needed in your virtual environment

```bash
pip install black==21.4b2
pip install flake8==3.9.1
pip install flake8-unused-arguments==0.0.6
pip install freezegun==1.1.0
pip install isort==5.8.0
pip install mypy==0.812
pip install pylint==2.8.2
pip install pytest==6.2.3
pip install pytest-cov==2.11.1
```

Once installed, the commands you need to run can be found in the `docker-compose.yml` file: https://github.com/apoclyps/reviews/blob/main/docker-compose.yml

The following is a list of commands used when linting ( each of these are required to pass before CI will pass).

```bash
isort .
black --line-length 88 --check .
flake8 .
mypy . --ignore-missing-imports
pylint --rcfile=.pylintrc reviews
pytest -cov=reviews -cov-report -vvv
```